application/controllers/gcp_survey_response.py

```diff
@@ -113,7 +113,7 @@ def put_file_into_gcp_bucket(self, message: dict):
         gnugpg_secret_keys = current_app.config['ONS_GNU_PUBLIC_CRYPTOKEY']
         ons_gnu_fingerprint = current_app.config['ONS_GNU_FINGERPRINT']
         encrypter = GNUEncrypter(gnugpg_secret_keys)
-        encrypted_message = encrypter.encrypt(message, ons_gnu_fingerprint)
+        encrypted_message = encrypter.encrypt(message['file'], ons_gnu_fingerprint)
```

Comment:
I don't think this will work because the file attribute of message is a base64 encoded representation of the file contents where we need the actual bytes